llvm/include/llvm/Frontend/Atomic/Atomic.h

```diff
@@ -12,44 +12,45 @@
 
 #include "llvm/ADT/DenseMap.h"
 #include "llvm/IR/DataLayout.h"
+#include "llvm/IR/IRBuilder.h"
 #include "llvm/IR/Instructions.h"
 #include "llvm/IR/Intrinsics.h"
 #include "llvm/IR/Module.h"
 #include "llvm/IR/Operator.h"
 #include "llvm/IR/RuntimeLibcalls.h"
```

Comment:
Maybe double check that you still need these includes in the header? E.g. I don't think you need RuntimeLibcalls.h?